box-browse-sdk/src/main/res/values/strings.xml

```diff
@@ -26,7 +26,7 @@
     <string name="box_browsesdk_search_results_header">Results in <![CDATA[<b>%1$s</b>]]></string>
 
     <string name="search_hint" description="hint text for Search View. Argument represents a folder name">Search in %1$s</string>
-    <string name="global_search_hint" description="hint text for top level Search View">Search in Box</string>
+    <string name="default_search_hint" description="Default hint text for Search View">Search</string>
```

Comment:
Should we reuse:
<string name="search" description="The search menu item description text">Search</string>
?